Examples/ExamplesApp/ExamplesApp.swift

```diff
@@ -26,12 +26,17 @@ struct ExamplesApp: App {
     init() {
         #warning("Require user to sign in with an ArcGIS identity or set your developer API key")
         /*
-         Use of Esri location services, including basemaps and geocoding, requires either an ArcGIS identity or an API Key. For more information see https://links.esri.com/arcgis-maps-sdk-security-auth.
-         1) ArcGIS identity: An ArcGIS named user account that is a member of an organization in ArcGIS Online or ArcGIS Enterprise.
-         2) API key: A permanent key that gives your application access to Esri location services. Create a new API key or access existing API keys from your ArcGIS for Developers dashboard (https://links.esri.com/arcgis-api-keys).
-         Production deployment of applications built with ArcGIS Maps SDK requires you to license ArcGIS Maps SDK functionality. For more information see https://links.esri.com/arcgis-maps-sdk-license-and-deploy.
+         Use of ArcGIS location services, such as basemaps, geocoding, and routing, requires either user authentication or API key authentication. For more information see https://links.esri.com/arcgis-maps-sdk-security-auth or https://links.esri.com/arcgis-maps-sdk-license-and-deploy.
+         1) User authentication: Allows users with an ArcGIS account to sign into an application and access secure services.
+         2) API key authentication: A long-lived access token that gives your application access to secure services. Go to the [Create an API key](https://developers.arcgis.com/documentation/security-and-authentication/api-key-authentication/tutorials/create-an-api-key/) tutorial to copy the API key access token. To access all services used by this application set the location services privileges to:
+         
+           Location services > Basemaps > Basemaps style service
```

Comment:
Can you update this to

Location services > Basemaps
Location services > Geocoding
Location services > Routing

This means that the text is correct for both ArcGIS Location Platform accounts and ArcGIS Online accounts. This was a recent change/update. Thanks

Reply:
I don't know what the toolkit can do. Only request that they set these privileges if they need them.